functions/src/api/v1/reminders/deleteReminder.ts

```diff
@@ -31,7 +31,5 @@ export const deleteReminder = wrap(async (req: Request, res: Response) => {
 
     await deleteReminderDb(reminderId)
 
-    res.send({
-        success: true,
-    })
+    res.send()
```

Comment:
(non-blocking), isn't `204 status` the most appropriate for a response with no content ?

Reply:
#350

Reply:
Created a ticket in case we have other spots where to fix this